src/Stratis.StratisD/Properties/launchSettings.json

```diff
@@ -1,7 +1,8 @@
 {
   "profiles": {
     "Stratis.StratisD": {
-      "commandName": "Project"
+      "commandName": "Project",
+      "commandLineArgs": "-testnet -addnode=127.0.0.2"
```

Comment:
You probably don't want to change this file

Reply:
correct, whoops, undoing

Reply:
Here we go again